tests/e2e/compare/output/markdownTable.ts

```diff
@@ -260,15 +263,13 @@ function markdownTable(table, options = {}) {
     sizeMatrix.splice(1, 0, sizes);
 
     rowIndex = -1;
-    /** @type {Array<string>} */
-    const lines = [];
+    const lines: Array<string> = [];
 
     while (++rowIndex < cellMatrix.length) {
         const row = cellMatrix[rowIndex];
         const sizes = sizeMatrix[rowIndex];
         columnIndex = -1;
-        /** @type {Array<string>} */
-        const line = [];
+        const line: Array<string> = [];
```

Comment:
Eslint should error here: 
```suggestion
        const line: string[] = [];
```